src/Oro/Bundle/ConfigBundle/front/src/feature/hooks/useSaveConfig.ts

```diff
@@ -0,0 +1,30 @@
+import {NotificationLevel, useNotify, useRoute, useTranslate} from '@akeneo-pim-community/shared';
+import {
+  configBackToFront,
+  configFrontToBack,
+  ConfigServicePayloadBackend,
+  ConfigServicePayloadFrontend,
+} from '../models/ConfigServicePayload';
+
+export function useSaveConfig() {
+  const __ = useTranslate();
```

Comment:
To avoid confusion with the use of the Loadash library or the `oro/translator` legacy module, we use to explicitly name the function like below.
 
```suggestion
  const translate = useTranslate();
```

Reply:
yes no problem